WordPress/UITestsFoundation/Screens/ReaderScreen.swift

```diff
@@ -74,6 +74,10 @@ public class ReaderScreen: ScreenObject {
         if backButton.isHittable { backButton.tap() }
     }
 
+    public func verifyReaderScreenLoaded() {
+        XCTAssertTrue(isLoaded)
+    }
```

Comment:
I noticed there are at least three usages of this pattern.

Have you considered abstracting it to make it available to all `ScreenObject` subclasses?

```swift
public extension ScreenObject {

    func assertIsLoaded(file: StaticString = #file, line: UInt = #line) {
        XCTAssertTrue(isLoaded, file: file, line: line)
    }
}
```

Adding `file` and `line` will make the test fail at the call site instead of within this method, which should provide clearer test failures at least from within Xcode.

Reply:
Thanks for this suggestion @mokagio! This pattern is being used across multiple tests and your recommendation makes sense, I tested it out locally and it works well. I did however rename it to `assertScreenIsLoaded` to be clearer that we're asserting the screen. Changes made in https://github.com/wordpress-mobile/WordPress-iOS/pull/20903/commits/ad42bf4654b9ce84d575cfb81fab74111c2925db

With the change, I could also remove all the `verifyScreenLoaded` from individual screens. I tested by making one test failed and the failure message is clear:
```
    t =    52.23s         Find the "Media Rows" Cell
    t =    52.41s         Capturing element debug description
/xxx/WordPress-iOS/WordPress/UITests/Tests/MainNavigationTests.swift:28: error: -[JetpackUITests.MainNavigationTests testLoadsPeopleScreen] : XCTAssertTrue failed
```

Reply:
TIL about `#file` and `#line`. =]

> I did however rename it to assertScreenIsLoaded to be clearer that we're asserting the screen.

That was a nice move. It helps avoiding confusion especially in the middle of longer tests.